README.md

```diff
@@ -6,6 +6,78 @@
 ![Screenshots with my stupid grinning face](images/screenshot.jpg)
 (Credits for the nice backgrounds to [Mary Sabell](https://dribbble.com/shots/4686178-Bauhaus-Poster) and [PhotoFunia](https://photofunia.com/effects/retro-wave))
 
+## Maintainers
+
+  * Phil Ashby (@phlash)
+  * Benny Baumann (@benbe)
+  * Florian Echtler (@floe)
+
+## Building
+
+Install dependencies (`sudo apt install libopencv-dev build-essential v4l2loopback-dkms curl`).
+
+Clone this repository with `git clone --recursive https://github.com/floe/backscrub.git`.
+To speed up the checkout you can additionally pass `--depth=1` to `git clone`.
+This is okay, if you only want to download and build the code, however, for development it is not recommended.
+
+Use `cmake` to build the project: create a subfolder (e.g. `build`), change to that folder and run: `cmake && make -j4` (please replace "4" with the number of your cores).
+
+Alternatively, you can also run `make` to build everything (should also clone and build Tensorflow Lite).
+
+## Usage
+
+First, load the v4l2loopback module (extra settings needed to make Chrome work):
+```
+sudo modprobe v4l2loopback devices=1 max_buffers=2 exclusive_caps=1 card_label="VirtualCam"
```

Comment:
I like to force the device numbering here, avoids confusion:
```bash
sudo modprobe v4l2loopback video_nr=10 label="Virtual Cam" max_buffers=2 exclusive_caps=1
```

Reply:
Maybe also document the configuration via `/etc/modprobe.d/v4l_loopback.conf` by setting up a configuration like:
```
# V4L loopback driver
options v4l2loopback max_buffers=2
options v4l2loopback exclusive_caps=1
# options v4l2loopback devices=4
options v4l2loopback card_label="VirtualCam 0,VirtualCam 1,VirtualCam 2,VirtualCam 3"
```

Doing so reduces the complexity to load the driver. Also adding `/etc/modules-load.d/v4l2loopback.conf` with the following content allows to auto-load the driver on every boot:
```
v4l2loopback
```